Add updates to FullStory cloud mode destination docs (#3917)
segmentio/segment-docs: src/connections/destinations/catalog/actions-fullstory-cloud/index.md

### Getting Started

1. You need a FullStory API Key to use the FullStory cloud mode destination. Refer to [this article](https://help.fullstory.com/hc/en-us/articles/360052021773-Managing-API-Keys){:target="_blank"} to learn how to generate a new API Key within FullStory.
2. From the Segment web app, click **Catalog**, then click **Destinations**.
3. Find “FullStory Cloud Mode (Actions)” in the Destinations list and click it.
4. Click **Configure FullStory Cloud Mode (Actions)**.
5. Select an existing Source to connect to FullStory Cloud Mode (Actions).
6. Provide a Destination Name and select **Fill in settings manually.** Ensure the “Actions” destinations framework is selected and click **Save.**
7. On the **Basic Settings** page, enter your FullStory API Key from step 1 and click **Save Changes**.
8. On the **Mappings** tab, you can view default mappings as well as add, modify, or disable mappings. Confirm that the "User ID" FullStory property is mapped to the ID previously used to identify the user. For more information, please refer to the [API documentation](https://developer.fullstory.com/server-events){:target="_blank"}.

> Please note: Events sent through Segment Cloud Mode count towards your FullStory server event quota. To see your company’s current quota allotment, view the Subscription information in your Account Settings within FullStory.

Reviewer: ```suggestion
> info ""
> Events that you send through to FullStory through a Cloud-mode connection count towards your FullStory server event quota. To see your company’s current quota allotment, view the Subscription information on the Account Settings page in FullStory.
```

Author: sorry for the delay, the suggestion has been applied. 👍 appreciate it.
